src/pkcs11/pkcs11-object.c

```diff
@@ -281,8 +296,15 @@ C_GetAttributeValue(CK_SESSION_HANDLE hSession,	/* the session's handle */
 		}
 	}
 
-out:	sc_log(context, "C_GetAttributeValue(hSession=0x%lx, hObject=0x%lx) = %s",
+out:
+	;	
+	const char* name = lookup_enum ( RV_T, rv );
```

Comment:
the variable definition can come only in the beginning of the block in the current C standard OpenSC is using.